.github/workflows/tests.yaml

```diff
@@ -4,17 +4,24 @@ on: [pull_request, create]
 jobs:
   build:
     name: Tests
-    runs-on: ubuntu-22.04
+    runs-on: ${{ matrix.runner }}
     strategy:
       fail-fast: false
+      matrix:
+        include:
+          - runner: ubuntu-22.04
+            coreboot-tests: true
```

Comment:
Do you mind doing something like `variant: coreboot` here and something like `variant: no coreboot` - will produce nicer actions titles - otherwise - nice use of matrix!